src/rust/engine/build_utils/src/tests.rs

```diff
@@ -0,0 +1,19 @@
+use crate::BuildRoot;
+
```

Comment:
Should these have our clippy headers?

Reply:
Only entry points to crates (normally, `lib.rs` or `bin.rs` files) need those; these are just modules :)

We have a linter installed which makes sure all of the correct files have them, too